src/utils/string.js

```diff
@@ -1,4 +1,7 @@
-/* @flow strict */
```

Comment:
Same as previous comment, please do not lower the strictness of the code.
If you are trying to solve a type issue, there is likely an alternative solution.
You can ask for help if you can't figure it out.